third_party/terraform/tests/data_source_google_kms_crypto_key_test.go

```diff
@@ -0,0 +1,70 @@
+package google
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccDataSourceGoogleKmsCryptoKey_basic(t *testing.T) {
+	kms := BootstrapKMSKey(t)
+	projectId := getTestProjectFromEnv()
+	projectOrg := getTestOrgFromEnv(t)
+	projectBillingAccount := getTestBillingAccountFromEnv(t)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceGoogleKmsCryptoKey_basic(projectId, projectOrg, projectBillingAccount, kms.KeyRing.Name, kms.CryptoKey.Name),
```

Comment:
This is closer to what I imagined using bootstrapKMS would be, but you can still exclude a ton of the resource building so that this test only focuses on the datasource functionality (which is test that an existing resource can be imported).